src/components/AllUsersModal/AllUsersModal.jsx

```diff
@@ -0,0 +1,240 @@
+import Modal from '@mui/material/Modal';
+import { useEffect, useState } from 'react';
+import { IoCloseOutline } from 'react-icons/io5';
+import URLs from '@/constants/constants';
+import { getUser } from '@/redux-store/selectors';
+import { useNavigate } from 'react-router-dom';
+import { useFetch } from '@/hooks/useFetch';
+import { useSelector } from 'react-redux';
+import { routesPath } from '@/routes/routesConfig.jsx';
+import { useChatContext } from '@/providers/ChatProvider';
+import { axiosClient } from '@/services/api';
+import { BiArrowBack } from 'react-icons/bi';
+import {
+  BoxStyled,
+  CloseBtn,
+  Item,
+  LetterAvatar,
+  SearchInput,
+  SendMessageBtn,
+  UsersList,
+  IconSearch,
+  SearchBox,
+  Subtitle,
+  Title,
+  AboutUser,
+  InfoIcon,
+  UserInfo,
+  ButtonBack,
+  MoreInfoBtn,
+  MainBoxStyled,
+  UserName,
+  AvatarInList,
+  AvatarInUserBlock,
+  UserInfoBox,
+  UserContact,
+  LetterAvatarInUserBlock,
+} from './AllUsersModalStyled';
+
+const AllUsersModal = ({ isOpen, onClose }) => {
+  const currentUserId = useSelector(getUser)?.id;
+  const [searchedValue, setSearchedValue] = useState('');
+  const [filteredUsers, setFilteredUsers] = useState([]);
+
+  const { dataUserChats, updateUserChats } = useChatContext();
+  const { responseData: users } = useFetch(URLs.getAllUsers, '');
```

Comment:
It’s unnecessary anymore to put quotation marks after URL.

Reply:
done